lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    RaiseHandStdDisabled: 'Bajar la mano',
+    RaiseHandEnable: 'Levantar la mano',
+    raiseHandNotify: 'Alumno(s) levantó la mano',
+    RaiseHandStdEnabled: 'Mano levantada',
+    readonlymode: 'Solo lectura',
+    receivedVotes: 'Votos recibidos',
+    reclaim: 'Reclamar',
+    Rectangle: 'Rectángulo',
+    rejected: 'Otra aplicación en su computadora podría estar usando su cámara web. Debe cerrar las demás aplicaciones',
+    reload: 'Recargar',
+    reloadDoc: 'Recargar documento',
+    removeContext: 'Eliminar marcador',
+    replay: 'Repetir',
+    Replay: 'Repetir desde el inicio.',
+    replay_message: 'Gracias por visualizarlo.',
+    requestScreenShare: 'Solicitar compartir pantalla',
+    requestedScreenShare: 'Se solicita compartir pantalla',
+    Reset: 'Restablecer',
+    Rtime: 'Tiempo restante',
+    rusureCquiz: '<span>¿Estás seguro de cerrar este cuestionario? </span>',
+    rvtu: 'Votos recibidos/ usuarios totales',
+    reply: 'Repetir',
+    safariBrowserIssue: 'Su navegador Safari {virtualclass1} no es compatible. Es aconsejable usar Chrome y Firefox',
+    Save: 'Grabar',
+    savesession: '¿Quieres guardar la sesión actual?',
+    savesessionTechSupport: "¿Quieres guardar la sesión actual? <br /> Una vez que descargue la sesión, las actualizaciones no estarán disponibles al volver a cargarla.",
+    screensharealready: 'La pantalla se está compartiendo.',
+    ScreenShare: 'Compartir pantalla',
+    screensharemsg: 'La pantalla de arriba se está compartiendo.',
+    screensharenotsupport: 'Pantalla compartida no soportada',
+    screensharedenied: 'El usuario rechazó compartir su pantalla',
+    second: 'Segundo',
+    selfview: 'Deshabilitar vista pública',
+    SetTimer: 'Establecer tiempo',
+    sesseionkeymissing: 'Falta la KEY de la interfaz', // from javascript
+    sentPackets: 'Enviar <br/><span>Paquetes</span>',
+    // 'sessionend': "Close session.",
+    SessionEnd: 'Sesión finalizada',
+    sessionexpired: 'La sesión ha expirado',
+    sessionendmsg: 'La sesión ha sido cerrada. Puede cerrar su navegador.',
+    setting: 'Configuración',
+    Searchuser: 'Buscar usuario',
+    sendMessage: 'Enviar mensaje',
+    share: 'Compartir',
+    ShareVideo: 'Compartir vídeo',
+    shareAnotherYouTubeVideo: 'Compartir otro vídeo de YouTube',
+    SharePresentation: 'Compartir presentación',
+    SharePresentationdbHeading: 'Panel de Presentación',
+    sharetoall: 'Habilitar vista pública',
+    Showresulttostudents: 'Mostrar resultados de los alumnos',
+    sitePoll: 'Encuesta nivel del centro',
+    Skip: 'Cerrar',
+    someproblem: 'Hubo algún problema al cargar el archivo. Inténtelo de nuevo.',
+    SourceUnavailableError: 'Por favor, asegúrese de que no se esté utilizando la misma webcam en otros navegadores o aplicaciones.',
+    // 'SourceUnavailableError' : 'Otra aplicación podría estar usando su webcam. Cierre las demás aplicaciones que puedan estar utilizándola.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 144. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)